wp_api/src/wordpress_org/plugin_directory.rs

```diff
@@ -328,4 +331,33 @@ mod tests {
         let parsed = serde_json::from_str::<PluginInformation>(json_string);
         assert!(parsed.is_ok(), "Failed to parse JSON: {:?}", parsed.err());
     }
+
+    #[test]
+    fn serialization_round_trip() {
+        let json_string =
+            include_str!("../../tests/plugin-directory/plugin-with-expected-types.json");
+        let plugin = serde_json::from_str::<PluginInformation>(json_string).unwrap();
+        let serialized = serialize_plugin_information(plugin).unwrap();
+        let deserialized = deserialize_plugin_information(serialized).unwrap();
+
+        let expected = serde_json::from_str::<PluginInformation>(json_string).unwrap();
+        assert_eq!(deserialized.name, expected.name);
+        assert_eq!(deserialized.author, expected.author);
```

Comment:
I think it'd be worth adding `PartialEq` and `Eq` to `PluginInformation` and its inner types, and then use:
```
assert_eq!(deserialized, expected);
```

I tested this locally and it worked as expected.